docs/zh-cn/1.3.6/user_doc/cluster-deployment.md

```diff
@@ -2,11 +2,11 @@
 
 # 1、基础软件安装(必装项请自行安装)
 
- * PostgreSQL (8.2.15+) or MySQL (5.7系列)：两者任选其一即可，如 MySQL 则需要 JDBC Driver 5.1.47+
- * [JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html) (1.8+)：必装，请安装好后在 /etc/profile 下配置 JAVA_HOME 及 PATH 变量
- * ZooKeeper (3.4.6+)：必装 
- * pstree or psmisc：Mac OS必装pstree，Fedora/Red/Hat/CentOS/Ubuntu/Debian必装psmisc
- * Hadoop (2.6+) or MinIO：选装，如果需要用到资源上传功能，可以选择上传到 Hadoop or MinIO 上
+* [PostgreSQL](https://www.postgresql.org/download/) (8.2.15+) or [MySQL](https://dev.mysql.com/downloads/mysql/) (5.7系列)：两者任选其一即可，如 MySQL 则需要 JDBC Driver 5.1.47+
```

Comment:
Thanks for picking the outdated PR @linjicong , it seems that we have already release versions 1.3.8, 1.3.9, 2.X.X above, could you please also change the content in those versions?